### PR TITLE
Update: extension icon

### PR DESCRIPTION
**Pull request by PikaDude**
Updates the extensions icon to something more unique and personal to the extension itself.

### CODE REVIEW

**Comment by PikaDude** [comment]
icon-new

File: options.html [hunk 1]
```diff
@@ -173,7 +173,7 @@ <h1>About</h1>
 
 				<div class="content">
 					<div class="image-container">
-						<img src="img/icon_32_kk.png" width="32" height="32" alt="Add-ons for NeoGAF">
+						<img src="img/icon_32.png" width="32" height="32">
```
Comment (jordigarcl) [inline options.html]: What about using the 128x128 version here? It does make a difference in my retina display

**Review by jordigarcl — APPROVED** [review]
Looking great

File: options.html (hunk 1 above)
Comment (PikaDude) [reply options.html]: Done in 38ee7b7

Comment (jordigarcl) [reply options.html]: <table><tr><td>
Screenshot 2020-01-17 at 11 59 48</td><td>
vs</td>
<td>
Screenshot 2020-01-17 at 12 00 00
</td></tr></table>

**Pull request merged**